README.md

```diff
@@ -0,0 +1,381 @@
+# Tariff Interpreter
+
+This Interpreter is a reference implementation for computing a JSON formatted bike-sharing tariff model. The tariff
+model is designed to be able to represent as many types of tariffs as possible and to have a modular structure. Thus, it
+is possible to calculate as many tariffs as possible with a single tariff interpreter and at the same time ensure a
+uniform format. In addition, a kind of receipt is issued at the end thanks to a flexible slot system. Thus, the
+composition of the travel costs is broken down as transparently as possible for the end customer.
+
+## Contents
+
+* [General Interpreter Operation](#general-interpreter-operation)
+* [Overview JSON Schemas](#overview-json-schema)
+
+## General Interpreter Operation
+
+This section will be added when the interface is built.
+
+## Overview JSON-Schema
+
+The modular tariff model is designed in such a way that there are three basic types of tariffs. When creating a tariff,
+you have to choose one. It is not possible to mix the tariffs. To decide which tariff is the right one, you can ask
+yourself the following questions:
+
+* Is my tariff distance dependent?
+
+  -> Then this model is unfortunately not the right one for you, because only tariffs that are time based can be
```

Comment:
I don't like these arrows :) But that's just my opinion. Do you want to remove them and just put the answers in a new line (`  ` to blanks after question)?
Questions in bold.